Misc/NEWS.d/next/Core and Builtins/2018-03-19-00-59-20.bpo-33083.Htztjl.rst

```diff
@@ -0,0 +1,3 @@
+math.factorial no longer accept arguments that are not int-like. A
```

Comment:
Sorry for nit-picking a bit. But 

"math.factorial no longer accepts arguments.." (accept -> accepts).

emmited -> emitted

Reply:
Thanks! :)